core/modules/filter/css/filter.css

```diff
@@ -91,3 +91,49 @@ input#edit-filters-filter-html-settings-allowed-html {
   display: inline-block !important;
   padding-bottom: 10px;
 }
+.image-library-choose-file .field-content {
+  font-size: 0.7rem;
+}
+.image-library-choose-file p {
+  margin: 0;
+}
+.image-library-image {
+  max-width: 150px;
+  max-height: 150px;
+}
+.library-view {
+  display:none;
+}
+.form-type-number .field-suffix {
+  display: inline;
+}
+.editor-dialog .form-item {
+  margin-bottom: 0.5em;
+}
+.editor-dialog .form-item .description {
+  margin-bottom: 0.5em;
+  margin-top: 0.2em;
+  padding: 0;
+  padding-left: 1em;
+  font-size: 0.9rem;
+}
+
+.image-library-choose-file {
+  float: left;
+  padding-left: 5px;
+  padding-right: 5px;
+  height: 170px;
+}
+
+.image-form-wrapper {
+  display: flex;
+}
+.image-form-left {
+  width:400px;
```

Comment:
Curious, why a fixed 400px? I thought it displayed better with relative % widths, eg 30% 70%.

Reply:
By using flex and a fixed LH column we can maximise the width and space available for images in the RH area, particularly relevant on large screens. I think maybe I will try reducing from 400 to nearer 300, whatever preserves the lavout within the LH area.

Reply:
Actually, I think the opposite. By applying a fixed width for the left it enforces that width on smaller screens at the expense of the right. I would recommend width: 30%, max-width: 300px for the left column and 70% for right.

Reply:
Well, I will try it next time I am using my large display. Perhaps @laryn has a preference?

Reply:
I think @docwilmot has a good suggestion with the left at width 30% and max-width 300px (and I'd add flex-grow: 0). The right can be 70% (and flex-grow: 1) so it can expand as needed.